cms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM taccwma/core-cms:latest
+FROM taccwma/core-cms:task-fp-1439-yearly-site-theme-for-texascale
```

Comment:
Branch/tag name is misleading. Changes are https://github.com/TACC/Core-CMS/pull/490.